### PR TITLE
Update replication task to include poste_deliveries

### PR DESCRIPTION
**Pull request by Sam-Hutchins**
Updating .yml file to include poste_deliveries in the replication task.

In order to service requests from the marketing team, we are adding this new table to the replication task.  

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by sureshc** [comment]
Can we use a Pull Request title that summarizes the change? The branch name looks ideal and could be used. :)

File: aws/dms/tasks.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ cron-pii:
         - name
         - session
 - pegasus.storage_apps
+- pegasus.poste_deliveries
```
Comment (sureshc) [inline aws/dms/tasks.yml:15]: Can we exclude the `contact_email` column? Or is there a need to use that data in our analytics systems.

Comment (LizGauthier) [reply aws/dms/tasks.yml:15]: there is a need for the contact_email col so I can match on users email for an adhoc regarding parent email sends

**Review by sureshc — APPROVED** [review]
Ping @code-dot-org/infrastructure when you’re ready for us to deploy this change!

**Comment by cat5inthecradle** [comment]
This change looks valid.

```
> bundle exec rake stack:data:validate RAILS_ENV=production
...
...
Modify DMSCronPii [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
```

Let us know when you're ready for this to go live. There is a CLI command that we need to run **after** you've merged this PR.

**Comment by cat5inthecradle** [comment]
Successfully deployed.

```
> bundle exec rake stack:data:start RAILS_ENV=production

...
...
 Stack update requested, waiting for provisioning to complete...
..2022-03-16 21:54:26 UTC- DMSCronPii [UPDATE_IN_PROGRESS]
.2022-03-16 21:54:27 UTC- DMSCron [UPDATE_IN_PROGRESS]
.......2022-03-16 21:55:13 UTC- DMSCron [UPDATE_COMPLETE]
2022-03-16 21:55:17 UTC- DMSCronPii [UPDATE_COMPLETE]
.2022-03-16 21:55:20 UTC- DATA-production [UPDATE_COMPLETE_CLEANUP_IN_PROGRESS]

Stack update complete.
Outputs:
```

**Pull request merged**